.github/workflows/merge_stage_test.yml

```diff
@@ -59,10 +43,8 @@ jobs:
       - name: Install PyTorch
         run: pip install torch==${{matrix.torch}}+cpu torchvision==${{matrix.torchvision}}+cpu -f https://download.pytorch.org/whl/torch_stable.html
       - name: Install MMCV
-        run: |
-          pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cpu/${{matrix.torch_version}}/index.html
-          python -c 'import mmcv; print(mmcv.__version__)'
-      - name: Install unittest dependencies
+        run: pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cpu/torch${MMCV_TORCH}/index.html
+      - name: Install other dependencies
```

Comment:
please install mmcv and mmengine  follow 
https://github.com/open-mmlab/mmsegmentation/blob/6cdc2c4a8a4f3abc71ee8acdd52c0a46326abe4c/.github/workflows/build.yml#L58-L66